src/components/timeline/TimelineMarkerCanvas.js

```diff
@@ -41,7 +41,8 @@ type Props = {
 const ROW_HEIGHT = 16;
 const TEXT_OFFSET_TOP = 11;
 const TWO_PI = Math.PI * 2;
-const MARKER_DOT_RADIUS = 0.25;
+const MARKER_MIN_DOT_RADIUS = 0.15;
```

Comment:
I like this radius difference, but the min dot radius feels a bit small for hovering over.